api/datastore/sql/migrations/18_create_new_calls.go

```diff
@@ -0,0 +1,49 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/fnproject/fn/api/datastore/sql/migratex"
+	"github.com/jmoiron/sqlx"
+)
+
+func up18(ctx context.Context, tx *sqlx.Tx) error {
```

Comment:
i'm not sure that this migration does exactly what we need.

what if routes table has lots of routes defined, they all must turn into function and triggers, unless we intentionally want to lose data.